official/docs/ruby/address/create_and_verify_address.rb

```diff
@@ -0,0 +1,16 @@
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_API_KEY']
+
+address = EasyPost::Address.create_and_verify(
```

Comment:
```suggestion
verified_address = EasyPost::Address.create_and_verify(
```

Once you fix this, the CI build should pass